docs/BLUEPRINTS.md

```
@@ -154,15 +154,15 @@ The `blueprints/index.json` file contains extracted metadata for UI display and
 1. Clone/download blueprints to `blueprints/` directory
 2. Run the import script:
   ```bash
-   python scripts/import_blueprints.py
+   python scripts/blueprints/import_blueprints.py
   ```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)